### PR TITLE
Added dao name to global state

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
Added dao name to global state

### CODE REVIEW

**Review by vuvth — APPROVED** [review]
Preapproval! 
Is the Dao name used for all proposals or will each proposal have a different name?

File: examples/dao/assets/dao-app-approval.py [hunk 1]
```diff
@@ -136,11 +136,13 @@ def scratch_proposal_active(idx: Int):
     max_duration = Bytes("max_duration")
     # a url with more information about the DAO
     url = Bytes("url")
+     # dao name
+    dao_name = Bytes("dao_name")
```
Comment (ratik21) [inline examples/dao/assets/dao-app-approval.py:140]: you're adding more storage to global state in contract. But you're not updating the global state schema during deployment.

File: examples/dao/scripts/deploy.js [hunk 2]
```diff
@@ -20,13 +20,15 @@ async function run(runtimeEnv, deployer) {
 	const minDuration = 1 * 60; // 1min (minimum voting time in number of seconds)
 	const maxDuration = 5 * 60; // 5min (maximum voting time in number of seconds)
 	const url = "www.my-url.com";
+	const daoName = "DAO";
```
Comment (ratik21) [inline examples/dao/scripts/deploy.js:23]: Is `daoName` different from `Dao App Name`?

File: examples/dao/test/dao-app-flow.js [hunk 3]
```diff
@@ -118,6 +119,7 @@ describe("DAO test", function () {
 			`int:${minDuration}`,
 			`int:${maxDuration}`,
 			`str:${url}`,
+			`str:${daoName}`,
```
Comment (ratik21) [inline examples/dao/test/dao-app-flow.js:122]: Please add an assertion in test as well

**Review by ratik21 — CHANGES_REQUESTED** [review]

File: examples/dao/assets/dao-app-approval.py (hunk 1 above)
Comment (vuvth) [reply examples/dao/assets/dao-app-approval.py:140]: I checked it. Number global state still is 2 slots.

Comment (Megha-Dev-19) [reply examples/dao/assets/dao-app-approval.py:140]: `globalBytes: 2` yes we have 2 slots

**Review by ratik21 — APPROVED** [review]

File: examples/dao/scripts/deploy.js (hunk 2 above)
Comment (Megha-Dev-19) [reply examples/dao/scripts/deploy.js:23]: yes, we use dao app name to store data in check point, while daoName is what we need to display on UI in sigma dao

**Comment by Megha-Dev-19** [comment]
> Preapproval! Is the Dao name used for all proposals or will each proposal have a different name?

All proposals will have same dao name

**Pull request merged**